### PR TITLE
Klaytn support

### PR DESCRIPTION
**Pull request by kalverra**
Adds Klaytn Support, and greatly simplifies adding Entries.

```go
Entry("Flux monitor suite on Simulated Network @simulated", networks.SimulatedEVM, big.NewFloat(10)),
Entry("Flux monitor suite on General EVM @general", networks.GeneralEVM(), big.NewFloat(.1)),
Entry("Flux monitor suite on Metis Stardust @metis", networks.MetisStardust, big.NewFloat(.01)),
Entry("Flux monitor suite on Sepolia Testnet @sepolia", networks.SepoliaTestnet, big.NewFloat(.1)),
Entry("Flux monitor suite on Klaytn Baobab @klaytn", networks.KlaytnBaobab, big.NewFloat(1)),
```

This cuts down on redundancy and lets us tailor funding for different networks with a wide variety of gas implementations. In a perfect world, we could accurately calculate the amount of funding needed for all test operations, but previous attempts at this have proven flakey at best. This more absolute approach has yielded much better results so far.

### CODE REVIEW

File: integration-tests/smoke/flux_test.go [hunk 1]
```diff
@@ -31,49 +31,11 @@ import (
...
-				chainlink.New(0, map[string]interface{}{
-					"env":      networks.SepoliaTestnet.ChainlinkValuesMap(),
-					"replicas": 3,
-				}),
-			),
+			Entry("Flux monitor suite on Simulated Network @simulated", networks.SimulatedEVM, big.NewFloat(10)),
+			Entry("Flux monitor suite on General EVM @general", networks.GeneralEVM(), big.NewFloat(.1)),
+			Entry("Flux monitor suite on Metis Stardust @metis", networks.MetisStardust, big.NewFloat(.01)),
+			Entry("Flux monitor suite on Sepolia Testnet @sepolia", networks.SepoliaTestnet, big.NewFloat(.1)),
+			Entry("Flux monitor suite on Klaytn Baobab @klaytn", networks.KlaytnBaobab, big.NewFloat(1)),
```
Comment (AnieeG) [inline integration-tests/smoke/flux_test.go:38]: Is there any specific way of calculating these values? can it be documented?

File: integration-tests/smoke/ocr_test.go [hunk 2]
```diff
@@ -89,17 +51,25 @@ var _ = Describe("OCR Feed @ocr", func() {
...
 			AddHelm(mockservercfg.New(nil)).
 			AddHelm(mockserver.New(nil)).
-			AddHelm(evmChart)
-		for _, chainlinkChart := range chainlinkCharts {
-			testEnvironment.AddHelm(chainlinkChart)
-		}
+			AddHelm(evmChart).
+			AddHelm(chainlink.New(0, map[string]interface{}{
+				"env":      testNetwork.ChainlinkValuesMap(),
+				"replicas": 6,
```
Comment (AnieeG) [inline integration-tests/smoke/ocr_test.go:71]: As opposed to previous implementation does it now always require 6 replicas?

File: integration-tests/smoke/vrfv2_test.go [hunk 3]
```diff
@@ -27,44 +27,11 @@ import (
...
-				}),
-				chainlink.New(0, map[string]interface{}{
-					"env": networks.SepoliaTestnet.ChainlinkValuesMap(),
-				}),
-			),
+			Entry("VRFv2 suite on Simulated Network @simulated", networks.SimulatedEVM, big.NewFloat(5)),
+			Entry("VRFv2 suite on General EVM @general", networks.GeneralEVM(), big.NewFloat(.05)),
+			Entry("VRFv2 suite on Metis Stardust @metis", networks.MetisStardust, big.NewFloat(.005)),
+			Entry("VRFv2 suite on Sepolia Testnet @sepolia", networks.SepoliaTestnet, big.NewFloat(.05)),
+			Entry("VRFv2 suite on Klaytn Baobab @klaytn", networks.KlaytnBaobab, big.NewFloat(.5)),
```
Comment (AnieeG) [inline integration-tests/smoke/vrfv2_test.go:34]: Again it would really help to understand how these gas values are derived

**Review by AnieeG — APPROVED** [review]

File: integration-tests/smoke/ocr_test.go (hunk 2 above)
Comment (kalverra) [reply integration-tests/smoke/ocr_test.go:71]: Ya, if you're running a basic OCR smoke test, you'll always do so with 6 replicas. If you want to use a different env setup for another test, you can just write another test, or make the environment a variable again.

**Review by tateexon — APPROVED** [review]

**Pull request merged**